tlslite/utils/cipherfactory.py

```diff
@@ -9,23 +9,23 @@
 from tlslite.utils import python_aesgcm
 from tlslite.utils import python_chacha20_poly1305
 from tlslite.utils import python_rc4
+from tlslite.utils import python_tripledes
 
 from tlslite.utils import cryptomath
 
-tripleDESPresent = False
+tripleDESPresent = True
+# 3DES cipher usage for backwards compatibility with older openSSL versions
```

Comment:
doc strings of variables/constants need to be strings (`"""`)

what older versions of OpenSSL?